pencil2d.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.1
+Type=Application
+Name=Pencil2D
+GenericName=Animation Software
+Comment=Create traditional hand-drawn animation using both bitmap and vector graphics
+Icon=pencil2d
+Exec=Pencil2D
```

Comment:
Alright I’ll do that. Just [keep in mind](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html#recognized-keys):

> The values […] should not be redundant with the values of Name or GenericName.

So I won’t add animation.